src/excalidraw-app/index.tsx

```diff
@@ -16,8 +20,16 @@ import { EVENT, LOCAL_STORAGE_KEY_COLLAB_FORCE_FLAG } from "../constants";
 import { loadFromFirebase } from "./data/firebase";
 import { restore } from "../data/restore";
 import { ExcalidrawImperativeAPI } from "../components/App";
+import { resolvablePromise } from "../utils";
 
 const excalidrawRef = React.createRef<ExcalidrawImperativeAPI>();
+
+// @ts-ignore
+excalidrawRef.current = {
+  readyPromise: resolvablePromise(),
+  ready: false,
+};
```

Comment:
at this point I'd migrate from `RefObject` to regular object because the ref gives us no benefit, except makes DX worse.